src/profiler_data.cc

```diff
@@ -5,8 +5,7 @@ namespace nodex {
   using v8::Context;
   using v8::Isolate;
 
-  ProfilerData::ProfilerData(v8::Local<Context> newContext, Isolate* isolate) {
-    context = newContext;
```

Comment:
Thanks for cleaning this up, at one point I was relying on `data->context` when I was using the v8 APIs instead of Nan.

Reply:
I have invited you to join this project :)